parser/sql/statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/statement/ddl/CreateTableStatement.java

```diff
@@ -36,7 +38,18 @@ public abstract class CreateTableStatement extends AbstractSQLStatement implemen
     
     private SimpleTableSegment table;
     
+    private SelectStatement selectStatement;
```

Comment:
Why move selectStatement to CreateTableStatement?

Reply:
Hi, mentor. When analyzing the SQL format of "CREATE TABLE AS SELECT" statement in MySQL, I found that Oracle, SQL Server, OpenGauss, and PostgreSQL all support similar SQL formats. The SelectStatement is currently defined in the SQLServerCreateTableStatement. Should we move the SelectStatement to the CreateTableStatement class for definition?

Reply:
Yes, commen segment should be moved to super class.